src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,11 @@
+$font-size-medium: 12px;
+$font-weight-less: 400;
```

Comment:
I was told this is the default value of font-weight, meaning you don't need to specify, it's like that by default

Reply:
Deleted.